src/lib/php/common-scheduler.php

```diff
@@ -45,7 +45,7 @@ function fo_scheduler_connect($IPaddr='', $Port='', &$ErrorMsg="")
     $IPaddr = '127.0.0.1';
   }
   if (empty($Port)) {
-    $Port = 5555;
+    $Port = 24693;
   }
   if (($sock = socket_create(AF_INET, SOCK_STREAM, SOL_TCP)) === false) {
     $ErrorMsg = "socket_create() failed: reason: " . socket_strerror(socket_last_error()) . "<br>\n";
```

Comment:
Is this a required change for this PR ??

Reply:
Not required but will be helpful.
I just stumbled across the code and thought it would be worth a fix to help someone working on it for the first time.

Reply:
Can we make it separately on other PR ? if it is fixing something ?

Reply:
I'll take this out as it does not fix anything for a running system, but it will save some time if they are running scheduler on a different port so I'll create a separate PR for it.